console/src/components/UI/Alert/Alert.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styles from './Alert.scss';
+const closeIcon = require('./images/x-circle.svg');
+const success = require('./images/check.svg');
+const info = require('./images/info.svg');
+const warning = require('./images/warning.svg');
+const error = require('./images/error.svg');
```

Comment:
Could we make it consistent and use `import` only?
Also, please add a new line to separate imports from the code.